gatsby/lobid/src/components/footer.html.js

```diff
@@ -3,48 +3,52 @@ import React from "react"
-              hbz
-            </a>
-          </span>
-          <span className="pull-right">
-            <a href="/warranty">
-              {this.props.warranty}
-            </a>
-            {" | "}
-            <a href="https://www.hbz-nrw.de/impressum">
-              {this.props.companyDetails}
-            </a>
-            {" | "}
-            <a href="https://github.com/hbz/lobid/blob/master/conf/Datenschutzerklaerung_lobid.textile">
-              {this.props.privacy}
-            </a>
-            {" | "}
-            <a
-              href="https://openbiblio.social/@lobid"
-              style={{ marginRight: "12px" }}
-            >
-              <i className="fa-brands fa-mastodon" aria-hidden="true"></i> Mastodon
-            </a>
-            <a
-              href="https://github.com/hbz/lobid"
-              style={{ marginRight: "12px" }}
-            >
-              <i className="fa-brands fa-github" aria-hidden="true"></i> GitHub
-            </a>
-            <a href="http://blog.lobid.org" style={{ marginRight: "12px" }}>
-              <i className="fa fa-pencil" aria-hidden="true"></i> Blog
-            </a>
-          </span>
+    render() {
+      return (
+        <div className="panel panel-default footer">
+          <div className="panel-body">
+            <span className="pull-left">
+              <img src={wappenPng} alt="NRW-Wappen" /> &nbsp; lobid |
+              ein Dienst des{" "}
+              <a href="https://www.hbz-nrw.de/produkte/linked-open-data">
+                hbz
+              </a>
+            </span>
+            <span className="pull-right">
+              <a href="/warranty">
+                {this.props.warranty}
+              </a>
+              {" | "}
+              <a href="https://www.hbz-nrw.de/impressum">
+                {this.props.companyDetails}
+              </a>
+              {" | "}
+              <a href="https://github.com/hbz/lobid/blob/master/conf/Datenschutzerklaerung_lobid.textile">
+                {this.props.privacy}
+              </a>
+              {" | "}
+              <a href="https://www.hbz-nrw.de/barrierefreiheit">
+                {this.props.accessibility}
+              </a>
+              {" | "}
+              <a
+                href="https://openbiblio.social/@lobid"
+                style={{ marginRight: "12px" }}
+              >
+                <i className="fa-brands fa-mastodon" aria-hidden="true"></i> Mastodon
+              </a>
+              <a
+                href="https://github.com/hbz/lobid"
+                style={{ marginRight: "12px" }}
+              >
+                <i className="fa-brands fa-github" aria-hidden="true"></i> GitHub
+              </a>
+              <a href="http://blog.lobid.org" style={{ marginRight: "12px" }}>
+                <i className="fa fa-pencil" aria-hidden="true"></i> Blog
+              </a>
+            </span>
+          </div>
```

Comment:
This seems to include formatting changes, makes it hard to see what actually changed.

Reply:
Oops, sorry. Only cleaned up indentation in order to check for correct closing tags more easily.